README.md

```diff
@@ -120,6 +128,18 @@ Pre-requisites
     ```
 
     </details>
+- Installing Android SDK emulator for Pixel 9.0
+    - ``` cd /Users/{{your user}}/Library/Android/sdk/cmdline-tools/latest/bin ```
+    - ``` ./sdkmanager --install "system-images;android-28;google_apis;x86" ```
+    - ``` echo "no" | ./avdmanager --verbose create avd --force --name "pixel_9.0" --device "pixel" --package "system-images;android-28;google_apis;x86" --tag "google_apis" --abi "x86" ```
+    - Run the emulator: ```./emulator @pixel_9.0 &```
+
+[9:28 AM] Guttman, Andrew
```

Comment:
@sebastianbarry did you intend to have this timestamp from andrew in here? What does it add?